cpp/lib/common/error_message.cpp

```diff
@@ -32,7 +32,7 @@ namespace TypeDB {
 
 namespace DriverError {
 
-#define ERR_DRIVER(ID, MSG) ERRMSG("CCL", "Driver Error", ID, MSG)
+#define ERR_DRIVER(ID, MSG) ERRMSG("CCCL", "Driver Error", ID, MSG)
```

Comment:
Yeah, I forgot about `DR` back then! Good find.

However, there are also `.ts` and `.py` drivers with codes like "DRI" (driver), "CON" (concept), etc. Do we want to change it as well?

Reply:
Well, you're right. I think that we can change all the drivers' errors to just one form of "DRI" instead of "{langpref}DR", it would be easier. If you agree with it, I'm going to do it.

Btw, by "conflicting" I meant logical conflicts, it was your argument when you initially asked me to refactor it... =)

Reply:
@flyingsilverfin @farost  The reason we introduced language prefixes in codes is so that we don't run into collisions between the Rust and the host language's error codes, mainly for internal errors. `INT02` coming from Java's ""Illegal state has been reached!" is distinct from Rust `INT02` "Unable to send response over callback channel (receiver dropped)."
We then reasoned that there's no guarantee that Rust driver won't be emitting its own Concept errors, e.g., so the prefix got added to all Java error codes. That should have also been the case in Python. Node is the only place where collisions are irrelevant as it's not using FFI.